backend/django_core/templates/anime/_layout.html

```diff
@@ -35,14 +35,24 @@
                             </search-form>
                         </div>
                     </div>
-                    <button class="avatar" aria-label="Avatar" aria-expanded="false">
-                        <figure
-                            class="avatar flex aspect-square text-surface-50 font-semibold justify-center items-center overflow-hidden isolate bg-surface-400-500-token w-12 md:w-[3.125vw] rounded-[0.4rem] md:rounded-[0.375vw]"
-                            data-testid="avatar"
-                        >
-                            <img class="avatar-image w-full h-full object-cover" style="" src="/images/Avatar.avif" alt="" />
-                        </figure>
-                    </button>
+                    <div class="dropdown dropdown-end flex">
+                        <div tabindex="0" role="button" class="avatar">
```

Comment:
conditional rendering here for `request.user.is_authenticated`
show `log in/ sign up` button